### PR TITLE
Sentry radial mode balance

### PR DESCRIPTION
**Pull request by Three333Digit**
## About The Pull Request
Purges power cell usage from sentries.
Activating radial mode on a sentry reduces its range by 2 tiles.

## Why It's Good For The Game
Radial mode is completely useless and a big noob trap. 

## Changelog
:cl:
balance: sentries no longer use power cells
balance: radial mode reduces the range of the sentry by 2 tiles
/:cl:

### CODE REVIEW

**Review by BraveMole — CHANGES_REQUESTED** [review]
Considering this is IIRC also making lasgun sentries have way more ammo, no.

Now, i agree that radial is a bit of a mess rn. What is suggest is removing baterries from sentries all together, and making radial have less range

File: code/modules/projectiles/sentries.dm [hunk 1]
```diff
@@ -54,6 +54,11 @@
 	spark_system.set_up(5, 0, src)
 	spark_system.attach(src)
 
+	if(!CHECK_BITFIELD(gun.turret_flags, TURRET_RADIAL))
+		range = gun.turret_range
+	else
+		range = gun.turret_range - 2
```
Comment (BraveMole) [inline code/modules/projectiles/sentries.dm:60]: This should be done l48, you are setting range twice. Also this should be using trinary ( ? operator)

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/sentries.dm [hunk 2]
```diff
@@ -45,7 +45,7 @@
 		iff_signal = id?.iff_signal
 
 	knockdown_threshold = gun.knockdown_threshold
-	range = gun.turret_range
+	range = !CHECK_BITFIELD(gun.turret_flags, TURRET_RADIAL) ? gun.turret_range : gun.turret_range - 2
```
Comment (BraveMole) [inline code/modules/projectiles/sentries.dm:48]:
```suggestion
	range = CHECK_BITFIELD(gun.turret_flags, TURRET_RADIAL) ?  gun.turret_range - 2 : gun.turret_range
```

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Review by BraveMole — APPROVED** [review]

**Comment by hyper2snyper** [comment]
this hurts my soul

**Review by BraveMole — APPROVED** [review]

**Pull request merged**